manifests/jetty.pp

```diff
@@ -110,6 +106,13 @@
     mode    => '0644',
     recurse => false,
     notify  => Class['shibboleth_idp::service'],
+  } ->
+
+  file { $idp_jetty_log_dir:
+    ensure => directory,
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+    mode   => '0644',
```

Comment:
I was mistaken. The new log dir is not recursive, so it should be a dir mode.
The $jetty_files, however are recursive so they should remain a file mode.